sites/public/src/pages/search-demo.tsx

```diff
@@ -0,0 +1,161 @@
+import React from "react"
```

Comment:
This is a separate page for now, but could easily be copied over into the main `listings.tsx` page